Libraries/Components/Touchable/Touchable.js

```diff
@@ -312,6 +314,15 @@ var TouchableMixin = {
     this.pressOutDelayTimeout && clearTimeout(this.pressOutDelayTimeout);
   },
 
+  propTypes: {
```

Comment:
PropTypes work but the website doesn't pick them up. Shall I duplicate the propType in the Touchable* and also leave it in the Touchable.Mixin?